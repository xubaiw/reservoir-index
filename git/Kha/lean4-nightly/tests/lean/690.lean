example (a b : Nat) (h : Nat.le a b) : Nat.le a (b + 1) := by
  induction h with
  | step x? hStep ih => admit
  | refl => admit

example (a b : Nat) (h : Nat.le a b) : Nat.le a (b + 1) := by
  induction h with
  | step hStep ih => trace_state; admit
  | refl => admit

example (a b : Nat) (h : Nat.le a b) : Nat.le a (b + 1) := by
  induction h with
  | @step x hStep ih => trace_state; admit
  | refl => admit
