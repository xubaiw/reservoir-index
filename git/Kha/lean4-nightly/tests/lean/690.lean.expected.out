690.lean:4:12-4:17: warning: declaration uses 'sorry'
690.lean:3:2-3:29: error: too many variable names provided at alternative 'step', #3 provided, but #2 expected
690.lean:3:24-3:29: warning: declaration uses 'sorry'
690.lean:9:12-9:17: warning: declaration uses 'sorry'
case step
a b m✝ : Nat
hStep : Nat.le a m✝
ih : Nat.le a (m✝ + 1)
⊢ Nat.le a (Nat.succ m✝ + 1)
690.lean:8:34-8:39: warning: declaration uses 'sorry'
690.lean:14:12-14:17: warning: declaration uses 'sorry'
case step
a b x : Nat
hStep : Nat.le a x
ih : Nat.le a (x + 1)
⊢ Nat.le a (Nat.succ x + 1)
690.lean:13:37-13:42: warning: declaration uses 'sorry'
